DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+  \addtohist{1}{2017-09-01}{Unreleased.}{John Parejko}
+}
+
+\makeatother
+
+\usepackage{xunicode}
+\begin{document}
+\maketitle
+\tableofcontents{}
+
+\section{Introduction}
+
+With deep astronomical images, it is extremely common that the relative
+astrometry and photometry between images is considerably more precise
+than the accuracy of external catalogs, where ``more precise'' can
+be as large as two orders of magnitude. For applications where the
+quality of relative astrometry is important or vital, it is important
+to rely on some sort of simultaneous astrometry solution, if possible
+optimal in a statistical sense.
+
+This package performs a least-squares fit to a set of images. Since
+it aims at statistical optimality, we maximize the likelihood of the
+measurements with respect to all unknown parameters required to describe
+the data. These parameters are mostly in two sets: the position on
+the sky of the objects in common between the images, and the mapping
+of each image to the sky. To these obvious parameters, one can add
+proper motions (where applicable), and parameters describing the differential
+effect of atmospheric refraction on the position of objects. It is
+clear that one cannot fit simultaneously the position on the sky and
+the mappings from CCD coordinates to the sky, without extra constraints:
+the ``sky'' coordinate system is then undefined, and one needs reference
+positions in order to fully define this frame. We use the GAIA catalog
+as our reference catalog, and may supplement it with deeper data (e.g.
+PS1) where available.
+
+SCAMP \footnote{See \href{http://www.astromatic.net/software/scamp}{http://www.astromatic.net/software/scamp}}is
+the reference package for simultaneous astrometry in astronomy, at
+least for relative alignment of wide-field images prior to stacking.
+Regarding optimization, SCAMP follows a somewhat different route from
+ours: it does not optimize over the position of common objects but
+rather minimizes the distance between pairs of transformed measurements
+of the same object. This approach is not a maximum likelihood optimization,
+and is likely statistically sub-optimal. The main drawback of SCAMP
+in the context of LSST is the fact that it is a program and not a
+library, and hence not flexible regarding formats of images and catalogs.
+But since SCAMP has been used for almost a decade in production by
+various teams, the quality checking tools it provides should likely
+be reproduced in the context of our package. We provide residual ntuples
+and hope that the first serious users will contribute plotting tools.
+
+Loading input catalogs and their metadata from disk is a large fraction
+of the total time. We can save time by re-using the input catalog
+to fit a similar style of multi-component model for the relative and
+absolute photometry.
+
+The plan of this note is as follows: we first sketch the algorithm
+(\S\ref{sec:algo}). We provide our least-squares formulation in
```

Comment:
Consider `\secref`.

Reply:
I've switched to using formatted references with prettyref, which handles it automatically.